addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -22,25 +22,25 @@
     &.o_masonry {
         &.o_spc-none div.o_masonry_col {
             padding: 0;
-            > img {
+            img {
```

Comment:
Why not `> img, > a > img` ?
Otherwise this means that any image being used inside it will receive that margin, eg you can drag & drop any inner snippet inside it like `Blockquote` which has an image, or basically anything and add an image yourself.
It is not that bad IMO since it is not the block purpose but if we can avoid it..